.circleci/config.yml

```diff
@@ -15,6 +15,12 @@ jobs:
       - run:
           name: Install Dependencies
           command: yarn --frozen-lockfile
+      - run:
+          name: Install Foundry
+          command: |
+            curl -L https://foundry.paradigm.xyz | bash || true
```

Comment:
Why the `|| true`? Does that mean we ignore failures?

Reply:
yup, that's correct! reason is that this installs installs Foundry but it's path is broken after the install due to this being in circleCI. the `|| true` means that we ignore the error that is produced but continue with the execution. Later, we configure the environment path to correctly make foundry accessible.  it's a bit janky...I'll admit... but it works.